### ISSUE TITLE
[BUG] Update Locator about mutable vs not mutable.

### ISSUE THREAD

**Issue by glennawatson** [reactiveui/splat#265]
Based on the question asked to StackOverflow here: https://stackoverflow.com/questions/54530075/splat-locator-current-vs-locator-currentmutable

**Comment by dpvreony** [comment]
made reference in the documentation to a possible interface change. but will need underlying changes and potentially the Setter capability moving into a SetLocator() method. Leave this as food for thought.

```cs
public interface IDependencyResolver : IReadonlyDependencyResolver, IMutableDependencyResolver
{
// remove the get methods
}


public interface IReadonlyDependencyResolver
{
// put get methods in here
}
```

**Comment by glennawatson** [comment]
Closed by #271
